serverless/README.md

```diff
@@ -46,91 +44,39 @@ Transform:
   - AWS::Serverless-2016-10-31
   - Name: DatadogServerless
     Parameters:
-      pythonLayerVersion: "<LAYER_VERSION>" # Use nodeLayerVersion for Node.js
       stackName: !Ref "AWS::StackName"
-      forwarderArn: "<FORWARDER_ARN>"
+      apiKey: "<DATADOG_API_KEY>"
```

Comment:
Do we want do specify that KMS is preferred?

Reply:
I'm hesitant to include KMS in the "happy path" because there are many other steps needed to make that work:
- Customer may want to set up a customer-managed key in KMS
- Customer will need to encrypt the API key in the AWS console for that Lambda. If this is the first deploy, you can't encrypt the key for a Lambda that doesn't exist yet.
- Customer will need to make sure that the Lambda functions' execution roles include permission to decrypt
Therefore, I think in the basic setup instructions we should document using the plaintext API key. We do have the KMS option documented below if users would prefer to use that.